simbad/command_line/__init__.py

```diff
@@ -70,6 +71,20 @@ def __init__(self):
         self.parse(tversion)
 
 
+def is_valid_file(parser, arg):
+    if not os.path.exists(arg):
+        parser.error("The file %s does not exist!" % arg)
+    else:
+        return arg
+
+
+def is_valid_dir(parser, arg):
+    if not os.path.isdir(arg):
```

Comment:
Swap these `if` statements, you always test for `if <something is correct> --- else` not if something is *not* true